### PR TITLE
QC for main sources of retirement income page

### PR DESCRIPTION
**Pull request by lhr-git**
### Description

List of proposed changes:

- changed structure of json files (removing link1, p1, paragraph-1, etc)
- changed structure of page to match QC
- added en images for the graphs

### Additional Notes

### CODE REVIEW

File: frontend/src/pages/learn/main-sources-of-retirement-income.tsx [hunk 1]
```diff
@@ -20,30 +22,42 @@ const MainSourcesOfRetirementIncome: FC = () => {
...
+        primary: t('learn-more.income-calculator.header'),
+        secondary: t('learn-more.income-calculator.description'),
       },
     ],
     [t]
   )
 
+  const mobile = useMediaQuery(theme.breakpoints.down('sm'))
+
+  const Img = ({ pension, classes }: { pension: string; classes?: string }) => (
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/main-sources-of-retirement-income.tsx:51]: Do not define components during render. React will see a new component type on every render and destroy the entire subtree’s DOM nodes and state. Instead, move this component definition out of the parent component “MainSourcesOfRetirementIncome” and pass data as props. `sonarlint(typescript:S6478)`

@Fbasham fixed this in other pages. Take a look with him.

File: frontend/public/locales/fr/learn/main-sources-of-retirement-income.json [hunk 2]
```diff
@@ -2,205 +2,323 @@
   "breadcrumbs": {
     "learn": "Apprendre"
   },
-  "meta": {
```
Comment (sebastien-comeau) [inline frontend/public/locales/fr/learn/main-sources-of-retirement-income.json:5]: `meta.description` key shouldn't be removed and it's already translated.

File: frontend/public/locales/en/learn/main-sources-of-retirement-income.json [hunk 3]
```diff
@@ -2,205 +2,323 @@
   "breadcrumbs": {
     "learn": "Learn"
   },
-  "meta": {
```
[the remark this answers is not in the document]
Comment (lhr-git) [reply frontend/public/locales/en/learn/main-sources-of-retirement-income.json:5]: it's actually further down the json, but the json is alphabetically sorted. It wasn't removed, just ... moved.

File: frontend/public/locales/fr/learn/main-sources-of-retirement-income.json (hunk 2 above)
Comment (lhr-git) [reply frontend/public/locales/fr/learn/main-sources-of-retirement-income.json:5]: it's actually further down the json, but the json is alphabetically sorted. It wasn't removed, just ... moved.

File: frontend/src/pages/learn/main-sources-of-retirement-income.tsx (hunk 1 above)
Comment (lhr-git) [reply frontend/src/pages/learn/main-sources-of-retirement-income.tsx]: Alright, I went ahead and removed it.

Comment (lhr-git) [reply frontend/src/pages/learn/main-sources-of-retirement-income.tsx]: Alright, alt text fixed.

**Review by sebastien-comeau — APPROVED** [review]
Fine now. :)

**Pull request merged**